packages/devtools/src/schema/types.ts

```diff
@@ -0,0 +1,82 @@
+import type { ModelAccountRelation, ModelDefinition } from '@ceramicnetwork/stream-model'
+import type { RuntimeViewField } from '@composedb/types'
+
+import type { ScalarSchema } from '../types.js'
+
+export type CreateParsedModelDefinition = {
+  action: 'create'
+  accountRelation: ModelAccountRelation
+  description: string
+  interface: boolean
+  implements: Array<string>
+  // relations: Record<string, unknown>
+  // views: ModelViewsDefinition
+}
+
+export type LoadModelDefinition = {
+  action: 'load'
+  id: string
+  // views: ModelViewsDefinition
+}
+
+export type ParsedModelDefinition = CreateParsedModelDefinition | LoadModelDefinition
```

Comment:
It feels really strange to me that every `CreateParsedModelDefinition` definition and every `LoadModelDefinition`  definition are `ParsedModelDefinition`s. 

My intuition would be that you need to parse a `LoadModelDefinition` or a `CreateParsedModelDefinition` to get a `ParsedModelDefinition`.

Do we already have a different `ModelDefinition` type somewhere and this is why you called this one `ParsedModelDefinition`? Can we streamline this somehow? It seems to me like we have lots and lots of new types in this PR and it's difficult to grasp what's what :).

Reply:
We also have
```
export type AbstractModelDefinition = CreateModelDefinition | LoadModelDefinition
```

, which is really confusing to me.

Reply:
Yes `ModelDefinition` is what gets exported by `@ceramicnetwork/stream-model`, it's already widely used in the codebase, and different from `ParsedModelDefinition`.
Parsed types and abstract types can be different, that's kind of the point of separating the parsing from the compiling logic, so they can be reasoned about independently.